Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -176,9 +198,9 @@ echo disk > /sys/power/state
 		while ($sw.elapsed -lt $timeout){
 			$vmCount = $AllVMData.Count
 			Wait-Time -seconds 15
-			$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password "date"
+			$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password -command "date"
```

Comment:
what is it? and when it will return 0?

Reply:
I expected to get the date from the VM, which means VM is accessible from the remote connection. This should be the indicator to move forward to the next steps. Before the time, this command got timeout from Run-LinuxCmd function, so it won't get any value. If it returns any value, it means that is the time remote connection is allowed. Should I add "; echo $?"?

Reply:
Fixed in 7dc196090095ce225ef3e8c7d80fc418701cc5e6

Reply:
I think we have a function that test port is accessible or not. Using this function, you have an assumption that, the VM can be accessed within timeout, otherwise it will throw exception then leave the test case ps1 script?

Reply:
If there is system hang for 15+ min, then it will be a time-out. And the next if-statement catch out the exception and run throw statement.  I also check out the VM status by PS1 command in the next step. If you post "that function" here, I can look into and find which way is better.

Reply:
@LiliDeng I am closing this PR and merging to the master now. If you still have a question, please let me know. I can certainly work on the change in case the existing function shows better logic when you send me the function name.